app/src/app.js

```diff
@@ -111,7 +113,9 @@ loader.loadRoutes(app);
 // In production environment, the port must be declared in environment variable
 const port = config.get("service.port");
 
-const server = app.listen(port, () => {});
+const server = app.listen(port, () => {
+  // do nothing for eslint
```

Comment:
can you move ``logger.info(`Server started in port:${port}`);`` to this function instead?